cmp/grid/columns/DatesTimes.js

```diff
@@ -30,6 +30,14 @@ export const dateTime = {
     width: 180
 };
 
+export const dateTimeSec = {
+    ...defaults,
+    align: 'left',
+    renderer: dateTimeSecRenderer(),
+    exportFormat: ExportFormat.DATETIME_FMT,
```

Comment:
The export format does not currently include second precision, as adding it changed the format in additional, undesired ways (uses 24-hr clock and adds timezone).

Reply:
No worries here - the nice thing about our Excel exports is that the real underlying value is baked into the spreadsheet, so you can adjust the formatting in Excel if desired to show seconds - the precision is there and available. 👍